src/olympia/signing/tests/test_views.py

```diff
@@ -959,6 +959,20 @@ def test_addon_blocked_guid_in_xpi(self):
         )
         assert response.status_code == 201
 
+    def test_deleted_webextension(self):
```

Comment:
This test (creating a new add-on, not a new version) was already passing with the existing code FYI, but I figured we might as well make sure.